AirCasting/ABConnector/ConnectingABViewModel.swift

```diff
@@ -19,16 +19,43 @@ class AirbeamConnectionViewModelDefault: AirbeamConnectionViewModel, ObservableO
     
     private let peripheral: CBPeripheral
     private let airBeamConnectionController: AirBeamConnectionController
+    private let userAuthenticationSession: UserAuthenticationSession
+    private let sessionContext: CreateSessionContext
     
-    init(airBeamConnectionController: AirBeamConnectionController, peripheral: CBPeripheral) {
+    init(airBeamConnectionController: AirBeamConnectionController,
+         userAuthenticationSession: UserAuthenticationSession,
+         sessionContext: CreateSessionContext,
+         peripheral: CBPeripheral) {
         self.peripheral = peripheral
         self.airBeamConnectionController = airBeamConnectionController
+        self.userAuthenticationSession = userAuthenticationSession
+        self.sessionContext = sessionContext
     }
     
     func connectToAirBeam() {
         self.airBeamConnectionController.connectToAirBeam(peripheral: peripheral) { success in
             self.isDeviceConnectedValue = success
             self.shouldDismissValue = !success
+            
+            if success {
+                self.configureABToFixed()
+            }
+        }
+    }
+    
+    private func configureABToFixed() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(2)) {
```

Comment:
wondering if this really needs to be done on a main queue. I guess for now yes, because this class is using main queue in general, but overall I think we need to switch to a private serial queue at some point. Both here and in BluetoothManager which pretty careless about threads too (I think all of our backbone code needs a solid threading diagnosis)